src/status_im/navigation/screens.cljs

```diff
@@ -343,12 +343,11 @@
      :component enter-seed-phrase/view}
 
     {:name      :screen/onboarding.enable-notifications
-     :options   {:theme                  :dark
-                 :layout                 options/onboarding-transparent-layout
-                 :animations             (merge
-                                          transitions/new-to-status-modal-animations
-                                          transitions/push-animations-for-transparent-background)
-                 :modalPresentationStyle :overCurrentContext}
```

Comment:
that's the problem, we should still show it overCurrentContext, because it's part of onboarding, and there should be blur

Reply:
@flexsurfer requested changes have been updated